plugins/org.wso2.developerstudio.eclipse.artifact.messagestore/src/org/wso2/developerstudio/eclipse/artifact/messagestore/provider/DatabaseTypeList.java

```diff
@@ -0,0 +1,38 @@
+package org.wso2.developerstudio.eclipse.artifact.messagestore.provider;
```

Comment:
Add licence header.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804